pkg/controller/master/master_controller.go

```diff
@@ -49,107 +49,107 @@ const (
 	maxRetries = 15
 )
 
-// NewMasterNodeController returns a new *MasterNodeController.
-func NewMasterNodeController(nodeInformer informers.NodeInformer, kubeClient kubeclientset.Interface, clusteroperatorClient clusteroperatorclientset.Interface) *MasterNodeController {
+// NewMasterController returns a new *MasterController.
+func NewMasterController(machineInformer informers.MachineInformer, kubeClient kubeclientset.Interface, clusteroperatorClient clusteroperatorclientset.Interface) *MasterController {
 	eventBroadcaster := record.NewBroadcaster()
 	eventBroadcaster.StartLogging(glog.Infof)
 	// TODO: remove the wrapper when every clients have moved to use the clientset.
 	eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: v1core.New(kubeClient.CoreV1().RESTClient()).Events("")})
 
 	if kubeClient != nil && kubeClient.CoreV1().RESTClient().GetRateLimiter() != nil {
-		metrics.RegisterMetricAndTrackRateLimiterUsage("clusteroperator_master_node_controller", kubeClient.CoreV1().RESTClient().GetRateLimiter())
+		metrics.RegisterMetricAndTrackRateLimiterUsage("clusteroperator_master_machine_controller", kubeClient.CoreV1().RESTClient().GetRateLimiter())
```

Comment:
This should probably be "clusteroperator_master_controller" or we should change the name of the controller to `MasterMachineController`.